addons/web_tour/static/tests/tour_service_tests.js

```diff
@@ -611,4 +611,122 @@ QUnit.module("Tour service", (hooks) => {
             assert.containsNone(target, ".o_tour_pointer");
         }
     );
+
+    QUnit.test("a failing tour logs the step that failed", async function (assert) {
+        patchWithCleanup(session, { tour_disable: false });
```

Comment:
Shouldn't be needed if you explicitly call startTour, the last 2 tests have it because they test tour auto-start

Reply:
indeed